### DISCUSSION TITLE
How to create a branch instead of forking?

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#5] [Q&A]
The step shown while editing the hometown CSV file? instead of forking?

**Comment by eculler** [comment]
Hi @DharaniSureshBabu,

Thank you for posting!

This is my bad -- I usually have students added as collaborators on a repository when we do this activity. The reason you can't add a branch is that you don't have permission. I've been thinking about how we want to do this, and so we are going to:

1. Do the hometowns activity with **forks**. Remember that y
2. Add your portfolio url to **this** repository using **branches**. To do this, we will need to collect everyone's GitHub username and add them as collaborators. We'll figure out how we want to do this soon.

You may want to do something similar with students so they can see the difference between forks and branches. Though we did this unintentionally, I think it's actually highlighting the difference between branches and forks really well (even it it's a bit frustrating)!

We'll update the textbook and videos to reflect this before next week.

@nquarder so you see the answer to this question -- I think it's going to be a common one!

**Reply by DharaniSureshBabu** [reply]
Great to know and sounds like a fantastic plan!
